contracts/src/v0.8/functions/dev/v1_X/FunctionsBilling.sol

```diff
@@ -96,7 +107,14 @@ abstract contract FunctionsBilling is Routable, IFunctionsBilling {
 
   /// @inheritdoc IFunctionsBilling
   function getDONFee(bytes memory /* requestData */) public view override returns (uint72) {
```

Comment:
Just to make it more readable, I think it might make sense to add unit names everywhere.
ie: `getDONFeeJuels()`

Reply:
I'm a little hesitant to do this because I believe it gets used in a lot of other places. I think the natspec documentation should be enough.